python/triton/testing.py

```diff
@@ -124,7 +148,12 @@ def do_bench(fn, warmup=25, rep=100, grad_to_none=None, quantiles=None, return_m
     end_event = di.Event(enable_timing=True)
     start_event.record()
     for _ in range(5):
-        cache.zero_()
+        if hasattr(cache, "zero_"):
```

Comment:
Seems to me that you can declare an interface and call something like `runtime.driver.active.clear_cache(cache)`